src/services/sessions.js

```diff
@@ -24,7 +27,9 @@ export async function getSession(params) {
 
 // queries all the available shared sessions from the database to display
 export async function getAllSessions() {
-  const endpoint = `${endpoints.pbench_server}/graphql`;
+  const endpoint = MOCK_UI
+    ? `${endpoints.pbench_server}/sessions/list`
+    : `${endpoints.pbench_server}/graphql`;
```

Comment:
Why not just usupr the `/graphql` endpoint for the mock'd environment as well?

Reply:
Because the old mocks depended on JSON payload; the new ones are strictly URI endpoints, which is a lot cleaner and more reliable!

Reply:
Correct! We would be able to remove all references to the `MOCK_UI` once everything routes through the `pbench_server` endpoint. This is temporary solution in order to allow us to use the mock environment as soon as possible. Ideally, we can specify a single endpoint pointing to the `pbench_server` regardless of being in a mocked environment or not.